src/device/cart/flashram.c

```diff
@@ -87,11 +87,11 @@ static void flashram_command(struct flashram* flashram, uint32_t command)
         break;
     case 0xe1000000:
         flashram->mode = FLASHRAM_MODE_STATUS;
-        flashram->status = 0x1111800100c20000LL;
+        flashram->status[0] = 0x11118001;
         break;
     case 0xf0000000:
         flashram->mode = FLASHRAM_MODE_READ;
-        flashram->status = 0x11118004f0000000LL;
+        flashram->status[0] = 0x11118004;
```

Comment:
Any chance you could name these or something?

Reply:
Unfortunately there is no datasheet for these flashram chips so I don't really know the meaning of these bits. However, I came across a "close cousin" (namely MX29L1611) which had some similarities with those used by big N. In its datasheet "0xc2" was the manufacturer code, plus @sanni (https://github.com/sanni/cartreader/blob/master/Cart_Reader/N64.ino#L1819) indicates that different products gets different answers. That's how I ended up with this PR with lower status bits for manufacturer code and product id.